src/test/java/org/jabref/logic/citationstyle/CitationStyleGeneratorTest.java

```diff
@@ -58,6 +57,22 @@ void aPACitation() {
         assertEquals(expected, citation);
     }
 
+    /**
```

Comment:
just mark the test @Disabled

Reply:
The comment is IMHO helpful. For someone diving into it 1 year later.

Reply:
Marked disabled and kept the comment.